content/en/containers/cluster_agent/admission_controller.md

```diff
@@ -128,13 +128,44 @@ Finally, run the following commands:
 - `kubectl apply -f agent-services.yaml`
 - `kubectl apply -f cluster-agent-deployment.yaml`
 
-### APM and DogStatsD
-
-To configure DogstatsD clients and APM tracers automatically, inject the environment variables `DD_AGENT_HOST` and `DD_ENTITY_ID` by using one of the following:
+### APM
+You can configure the Cluster Agent (version 7.39 and higher) to inject APM tracing libraries automatically.
```

Comment:
7.39+ only applies to the library injection, not the pod mutation for unified service tagging right?

Reply:
Any reason to try to fit everything in the same section?

Reply:
**+1 for having separate sections!**
Auto-instrumentation Vs. `DD_AGENT_HOST` and `DD_ENTITY_ID` injection are different features, they have different requirements and purposes and this should be clear in our docs.